docs/dev/index.md

```diff
@@ -239,6 +239,60 @@ class Solid {
 module.exports = Solid
 ```
 
+## Post Processing
+Custom post processing, such as pixel shaders, are handled inside of sketches. Hedron's post processing system is a thin wrapper around the [postprocessing](https://github.com/vanruesc/postprocessing) library, so it's a good idea to understand how that works. Multiple passes can be added to the rendering composer using `initiatePostProcessing` and things are updated every frame using `updatePostProcessing`.
+
+### Simple example
+Below is a simple example of how to achieve a bloom effect. A config file is also needed, which is exactly the same as a normal sketch config file.
```

Comment:
Good thinking! Although the reason I chose the bloom is because its simple to implement via the `postprocessing` library. Either way, having some bare bones example like invert definitely makes sense.